examples/source/components/amp-3d-gltf.html

```diff
@@ -10,7 +10,7 @@
 <html ⚡>
 <head>
   <meta charset="utf-8">
-  <link rel="canonical" href="/components/amp-3d-gltf/">
+  <link rel="canonical" href="<% hosts.platform %>/documentation/examples/components/amp-3d-gltf.html">
```

Comment:
Yes that makes more sense. The we would need to do some URL-Rewriting for request to the sample doc, because this one currently is really accessible by `/documentation/examples/components/amp-3d-gltf.html` or did you change this already?

Reply:
it's more than URL rewriting as links should also use the correct format.

Reply:
Done so by d62330d, 2b7ce47 and d5616d9 together with the discussed change to have a directory for each sample to hold a JS file that might contain logic.